docs/source/examples/vqe-demo_custom-gate.myst

```diff
@@ -0,0 +1,282 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.3
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Variational Quantum Eigensolver improved with Mitiq
+In this example we investigate how Mitiq can help reduce errors
+and improve convergence of a simple VQE problem executed
+on a simulated noisy backend.
+
+
+The pyQuil VQE example shown here is adapted from
+https://grove-docs.readthedocs.io/en/latest/vqe.html#, 
+and the VQE function in Grove.
+
+```{code-cell} ipython3
+import numpy as np
+from pyquil import get_qc, Program
+from pyquil.gates import RX, RY, S, T, Z, CNOT, MEASURE
+from pyquil.paulis import PauliTerm, PauliSum, sZ
+from pyquil.noise import pauli_kraus_map, append_kraus_to_gate
+from typing import List, Union
+from collections import Counter
+from matplotlib import pyplot as plt
+from scipy import optimize
+import mitiq
+from mitiq import zne
+from mitiq.zne.scaling.folding import fold_gates_from_right
+```
+
+Use the get_qc command to initialize the simulated backend 
```

Comment:
```suggestion
Use the `get_qc` command to initialize the simulated backend 
```
it would actually be even nicer to use Intersphinx and see if it works on the Examples' part of the documentation, to automatically link to the relevant part of the PyQuil docs. Here is an example of its use 
https://mitiq.readthedocs.io/en/stable/guide/guide-executors.html#qiskit-density-matrix-simulation-with-noise-and-sampling for the `execute_with_shots_and_noise()` function. [Link to github raw file](https://github.com/unitaryfund/mitiq/blob/51a10d1defffdd050e881fa5070c2f5484a4d6b9/docs/source/guide/guide-executors.rst#qiskit-density-matrix-simulation-with-noise-and-sampling).